### PR TITLE
Add gnali wrapper

### PR DESCRIPTION
**Pull request by xialiu71**
* Add wrapper for gnali
* Add a test and necessary test data

gnali repo can be found [here](https://github.com/phac-nml/gnali/tree/release/0.1)

### CODE REVIEW

File: tools/gnali/gnali.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<tool id="gnali" name="Get nonessential, LoF variants" version="0.1.0" python_template_version="3.6">
```
Comment (ericenns) [inline tools/gnali/gnali.xml]: I would change name to gNALI.

You can can a `<decscription></description>` section just inside and put what you have currently in the name field.

File: tools/gnali/.gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+*output*
```
Comment (Takadonet) [inline tools/gnali/.gitignore]: Please remove this file since we already have one in root of this repo.

Comment (xialiu71) [reply tools/gnali/.gitignore]: 2a7404f Removed .gitignore

File: tools/gnali/gnali.xml (hunk 1 above)
Comment (xialiu71) [reply tools/gnali/gnali.xml]: 2a7404f Changed name to "gNALI" and added description

**Comment by ericenns** [comment]
Hi Xia, I think everything looks good so you can resolve the WIP status by removing it from the title.

File: tools/gnali/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+categories: [Variant Analysis]
+description: A tool to find nonessential, loss-of-function gene variants
+name: gnali
+owner: nml
+long_description: Find nonessential, high-confidence loss-of-function gene variants based on the GRCh37 dataset.
+remote_repository_url: https://github.com/phac-nml/gnali/tree/release/0.1
```
Comment (ericenns) [inline tools/gnali/.shed.yml]: The links to `remote_repository_url` and `homepage_url` should likely just be the root of the github project.

Comment (xialiu71) [reply tools/gnali/.shed.yml]: 110aac0 Fixed URLs in `.shed.yml` and `gnali.xml`

File: tools/gnali/gnali.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    <requirements>
+        <requirement type="package" version="0.1">gnali</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        gnali -i '$test_genes' -o output
+    ]]></command>
+    <inputs>
+        <param type="data" name="test_genes" label="Test genes" format="txt" help="Specify a list of genes as HGNC symbols, separated by newline characters" />
+        <param type="select" name="database" label="Database" format="txt" help="Database to query" >
+            <option value="gnomad2.1.1">gnomAD2.1.1 (GRCh37/hg19)</option>
```
Comment (ericenns) [inline tools/gnali/gnali.xml]: can you set this as the default option

Comment (xialiu71) [reply tools/gnali/gnali.xml]: cb88a86 Made gnomAD2.1.1 the default database

File: tools/gnali/gnali.xml [hunk 5]
```diff
@@ -0,0 +1,77 @@
+<tool id="gnali" name="gNALI" version="0.1.0" python_template_version="3.6">
+    <description>Get nonessential, LoF variants</description>
+    <requirements>
+        <requirement type="package" version="0.1">gnali</requirement>
```
Comment (bgruening) [inline tools/gnali/gnali.xml]: @xialiu71 @Takadonet this should be 0.1.0.

Our bot tried to created a container for it but failed.

**Pull request merged**